packages/teams-js/src/internal/utils.ts

```diff
@@ -37,6 +37,22 @@ function validateHostAgainstPattern(pattern: string, host: string): boolean {
   return false;
 }
 
+export function getDomainsFromCDN(): void {
```

Comment:
nit: would be good to rename this so it doesn't sound like it returns some values. Open to naming suggestions if you have them, maybe something like `retrieveDomainsFromCDNAndStore`